resources/nl-core/ext-Category.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="ext-Category" />
```

Comment:
Would a name along the lines of "ext-AdditionalCategory" not be more appropriate?

Reply:
Agreed!